src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -215,6 +215,16 @@ class DefaultStorageKeyAllocator
     {
         return StorageKeyName::Formatted("f/%x/e/%x/sc/%x", fabric, endpoint, idx);
     }
+    // Operational State
+    static StorageKeyName OperationalStateOpStateList(EndpointId endpoint, ClusterId clusterId)
+    {
+        return StorageKeyName::Formatted("g/ops/sl/%x/%" PRIx32, endpoint, clusterId);
```

Comment:
This is forcing operational states to be in storage, as opposed to being defined by the application.  Why?